color/src/lib.rs

```diff
@@ -84,12 +84,13 @@
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
 #![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
 
+pub mod cache_key;
 mod color;
 mod colorspace;
+mod dynamic;
 mod gradient;
 mod missing;
 // Note: this may become feature-gated; we'll decide this soon
-mod dynamic;
```

Comment:
Not sure which mid the comment above this referred to… but if it was dynamic, it should move with it?

Reply:
Well caught! This is a casualty of rustfmt, unsurprisingly.

Reply:
Actually, this isn't the first time it's been a casualty of rustfmt, see - https://github.com/linebender/color/pull/12/files#diff-4c57624c2719867d1070e79f01066369fa55f94eec03f36b4b2e0ecf410e190aR28

I'll restore it to the original place. Although we really should probably just remove the comment